python/dgl/convert.py

```diff
@@ -912,31 +1102,248 @@ def _batcher(lst):
                     raise DGLError('Not all edges have attribute {}.'.format(attr))
             g.edata[attr] = F.copy_to(_batcher(attr_dict[attr]), g.device)
 
-    return g
+    return g.to(device)
+
+def bipartite_from_networkx(nx_graph,
```

Comment:
from_bipartite_networkx?

Reply:
I think `bipartite_from_networkx` is better.